### ISSUE TITLE
Support parameters to the Delivery deliverable

### ISSUE THREAD

**Issue by aaronpowell** [aaronpowell/Chauffeur#38]
If you're using the `delivery` Deliverable you can run into a problem with changing the password - you can't pass it in, it has to be in the `.delivery` file.

This is an issue when it comes to source control, you have to check in your passwords, which obviously isn't ideal.

Instead you should be able to pass in parameters when calling `chauffeur.runner.exe delivery`, something like:

```ps
PS> .\chauffeur.runner.exe delivery -p:Foo=bar -p:Baz=123
```

Inside the `.delivery` files you can then reference the parameters by name:

```
install -y
user change-password admin $Foo
```

**Comment by aaronpowell** [comment]
Decided to change it so that you have to wrap the token in `$`, not just prefix it. Makes finding tokens simpler.

Usage would now we:

```
install y
user change-password admin $foo$
```

CLI argument is unchanged

**Comment by aaronpowell** [comment]
And shipped in release [0.8.0](https://www.nuget.org/packages/Chauffeur/0.8.0)
